doc/changes/latest.inc

```diff
@@ -23,6 +23,9 @@ Current (1.1.dev0)
 
 Enhancements
 ~~~~~~~~~~~~
+
+- Add ``mne-icalabel`` to :func:`mne.sys_info` (:gh:`10615` by `Adam Li`_)
```

Comment:
the `:newcontrib:` should go at the top, can you move this down?

Reply:
I'll do this in #10592 to save one of us a rebase

Reply:
Ah thanks!